addons/test_mail/tests/test_performance.py

```diff
@@ -817,6 +817,51 @@ def test_complex_tracking_assignation(self):
         self.assertEqual(rec1.message_ids[1].notified_partner_ids, self.partners)
         self.assertEqual(len(rec1.message_ids), 2)
 
+    @mute_logger('odoo.tests', 'odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('employee')
+    @warmup
+    def test_complex_tracking_compute_fields(self):
+        # create some dummy partner titles to be able to test our compute method queries
+        partner_titles = self.env['res.partner.title'].create([
+            {'name': 'Title 1'},
+            {'name': 'Title 2'},
+        ])
+        self.customer.write({'title': partner_titles[0].id})
+        self.partners[0].write({'title': partner_titles[1].id})
+
+        with self.assertQueryCount(employee=9):
+            rec = self.env['mail.test.track.compute'].create({
+                'description': 'Description',
+                'partner_id': self.customer.id,
+            })
+            self.flush_tracking()
+
+        self.assertEqual(
+            rec.sudo().message_ids.tracking_value_ids,
+            self.env['mail.tracking.value']
+        )
+
+        self.env.invalidate_all()
+        with self.assertQueryCount(employee=10):
```

Comment:
We could even have several records and directly test in batch mode (could be funny).